README.md

```diff
@@ -44,3 +39,7 @@ Please take a look at existing pages to make sure new page follow the same struc
 
 4) Create a pull request for the new page.
 
+# Copyright Notice
+All contect in this repository is copyright ® of SimpleLogin SAS.
+
+**Copyright ® 2019 - 2023 SimpleLogin SAS. All rights reserved.**
```

Comment:
I wouldn't put the year here as we might forget to update it in 2 years :).

Reply:
Could be done. Altho, i don't now if it will legally still be valid then.

Reply:
I think the documentation doesn't really need a copyright so that should be ok.